doc/Streaming.xml

```diff
@@ -1146,19 +1146,21 @@
             <listitem><para>In all RTSP SETUP responses, a server should include the Timeout value according to [RFC 2326]
                 Section 12.37.</para></listitem>
             <listitem><para>A server shall interpret any RTSP request other than TEARDOWN as a keep-alive.</para></listitem>
-            <listitem><para>Clients should use SET_PARAMETER as keep-alive method for the RTSP session.</para></listitem>
             <listitem><para>A server shall interpret any RTCP receiver report for a unicast stream as keep-alive for the
                 RTSP session.</para></listitem>
             <listitem><para>If an RTSP session times out, a server should close related RTP streams as long as they are
                 not shared with other active RTSP sessions.</para></listitem>
           </itemizedlist>
-              <para>Note: for timeout handling see also the following sections of RFC 7826:</para>
-              <itemizedlist>
-                <listitem><para>Section 10.5. Showing Liveness</para></listitem>
-                <listitem><para>Annex C.1.6.2. RTSP Session Keep-Alive</para></listitem>
-              </itemizedlist>
-            <figure>
-              <title>Keep Alive</title>
+          <para>Note: for timeout handling see also the following sections of RFC 7826:</para>
+          <itemizedlist>
+            <listitem><para>Section 10.5. Showing Liveness</para></listitem>
+            <listitem><para>Annex C.1.6.2. RTSP Session Keep-Alive</para></listitem>
+          </itemizedlist>
+              <para><xref linkend="keepAlive"/> depicts the timing requirement for a client to signal being alive. The example below uses the RTSP command SET_PARAMETER. 
```

Comment:
Hi Hans,

For this statement "The example below uses the RTSP command SET_PARAMETER. "

 old image5.svg file shows RTSP command SET_PARAMETER for keep alive. New image5.svg in the PR just say RTCP or RTSP Command.